tests/has/devfile_source.go

```diff
@@ -132,99 +127,4 @@ var _ = framework.HASSuiteDescribe("devfile source", func() {
 		Expect(err).NotTo(HaveOccurred(), "Failed component pipeline %v", err)
 	})
```

Comment:
Are some of the specs in this test file and `e2e-demos/quarkus_curated.go` identical? If so, would it make sense to avoid running both `has` and `e2e-demos` test suites per PR in `e2e-tests` repo and `infra-deployments`?
(E.g. with the `--skip-file` flag.)

Reply:
@psturc , are different suites now. has tests are covering only application service and e2e-demo cover from app creation to gitops deployment

Reply:
I get that `has` suite is covering only application-service.

The thing I'm thinking about is what's the point of running the `has` suite in `e2e-tests` and `infra-deployments` PR tests, since it is running exactly the same specs as in `e2e-demo` suite:
https://github.com/redhat-appstudio/e2e-tests/blob/0743179d5730d31345a4add8781474aa71a43ea6/tests/has/devfile_source.go#L74-L128
https://github.com/redhat-appstudio/e2e-tests/blob/0743179d5730d31345a4add8781474aa71a43ea6/tests/e2e-demos/quarkus_curated.go#L77-L131

The spec `Wait for component pipeline to be completed` is quite time consuming, so I'm wondering whether we could avoid running this same spec twice (in `e2e-tests` PRs and `infra-deployments` PRs).

Reply:
we can skip in infra-deployments the has and have only the e2e-demo. WDYT?

Reply:
After talking to @flacatus we also agreed to remove [this spec](https://github.com/redhat-appstudio/e2e-tests/blob/586f79eba0d95db69275c749205a0d9d81fcdeb1/tests/has/devfile_source.go#L110-L128) from the HAS suite (since builds/pipelineruns are now managed by [build-service](https://github.com/redhat-appstudio/build-service/blob/80dca12a0186397b1fbd357ad2315a6bf8f8cd4f/controllers/component_build_controller.go#L152))